src/pylyrics/extract_lyrics.py

```diff
@@ -0,0 +1,219 @@
+# author: Macy Chan
+# date: 2022-01-14
+
+import requests
+import os, sys
+import pandas as pd
+import json
+import urllib.parse
+import re
+import lyricsgenius
+from alive_progress import alive_bar
+
+
+class HiddenPrints:
+    def __enter__(self):
+        """
+        Suppress printing output
+        """
+        self._original_stdout = sys.stdout
+        sys.stdout = open(os.devnull, "w")
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        """
+        Exit suppress printing output
+        """
+        sys.stdout.close()
+        sys.stdout = self._original_stdout
+
+
+def get_lyrics_from_Genius(genius, title, artist):
+    """
+    Find lyrics of song using Genius API
+
+    Parameters
+    ----------
+    title : string
+        song title
+    artist : string
+        artist of song
+
+    Returns
+    ----------
+    lyrics : string
+        lyrics of song
+    """
+    with HiddenPrints():
+        song = genius.search_song(title, artist)
+        if song:
+            lyrics = song.lyrics
+            return lyrics
+
+
+def extract_lyrics(token, songs):
```

Comment:
If we have this function here, maybe I need to take out the for loop in the plot_cloud function.

Reply:
Sorry I forgot I just need to extract one song instead of five. I will edit it.